tests/e2e/config/bootstrap.js

```diff
@@ -122,6 +122,40 @@ function observeConsoleLogging() {
 	} );
 }
 
+/**
+ * Runs Axe tests when the block editor is found on the current page.
+ *
+ * @return {?Promise} Promise resolving once Axe texts are finished.
+ */
+async function runAxeTestsForBlockEditor() {
+	if ( ! await page.$( '.block-editor' ) ) {
+		return;
+	}
+
+	await expect( page ).toPassAxeTests( {
+		// Temporary disabled rules to enable initial integration.
```

Comment:
Too bad this hasn't got traction so far: https://github.com/WordPress/gutenberg/issues/15452

Reply:
I am not sure what to do here. I was going to remove some of these rules, however part of the gutenberg editor fail, which gives a false positive. Unfortunately we have to stick with what gutenberg is doing, unless we want our tests to fail everytime we update upstream. :(